PySDM/physics/isotope_kinetic_fractionation_factors/pierchala_et_al_2022.py

```diff
@@ -0,0 +1,15 @@
+"""
+kinetic fractionation factor and epsilon kinetic
```

Comment:
OK to mention that it is used in Pierchala et al., but class would best be named with the label of the paper it was introduced in